.changeset/rich-crabs-love.md

```diff
@@ -0,0 +1,15 @@
+---
+"@turnkey/http": minor
+---
+
+Added `name` field to constructor. `isHttpClient` now uses this new field to complete the check
```

Comment:
I would add a wee bit more context here, explaining that if you were using `isHttpClient` before, you could see different behavior (though it's very unlikely)